README.md

```diff
@@ -6,20 +6,22 @@ Please note that `teleporter-token-bridge` is still under active development and
+Each bridge instance consists of one "hub" contract and at least one but possibly many "spoke" contracts. Each hub contract instance manages one asset to be bridged out to spoke instances. The hub contract lives on the Subnet where the asset to be bridged exists and locks that asset as collateral to be bridged to other Subnets. The spoke contracts, which each have a single specified hub contract, live on other Subnets that want to import the asset bridged by their specified hub. The token bridges are designed to be permissionless: anyone can register compatible spoke instances to allow for bridging tokens from the hub Subnet to that new spoke. The token bridge hub contract keeps track of token balances bridged to each spoke instance, and handles returning the original tokens back to the user when bridged back to the hub instance. Spoke instances are registered with their hub contract via a Teleporter message upon creation.
+
+Hub contract instances specify either an ERC20 token or the native token to be bridged, and allow for transferring the token to any registered spoke instances. The token representation on the spoke chain can also either be an ERC20 or native token, allowing users to have any combination of ERC20 and native tokens between hub and spoke chains:
 
 - `ERC20` -> `ERC20`
 - `ERC20` -> `Native`
 - `Native` -> `ERC20`
 - `Native` -> `Native`
 
-The destination tokens are designed to by default have compatibility with the token bridges on the source chain, and allow any custom logic to be implemented in addition. For example, developers can inherit and extend the destination ERC20 token contract to add additional functionality, such as a custom minting, burning, or transfer logic.
+The spoke tokens are designed to by default have compatibility with the token bridge on the hub chain, and allow custom logic to be implemented in addition. For example, developers can inherit and extend the `ERC20TokenSpoke` contract to add additional functionality, such as a custom minting, burning, or transfer logic.
 
-The token bridge also supports "multi-hop" transfers, where tokens can be transferred between destination chains. The multi-hop transfer first transfers the token from the origin destination chain to the home chain, where token balances are updated, and then triggers a second transfer to the final destination chain.
+The token bridge also supports "multi-hop" transfers, where tokens can be transferred between spoke chains. The multi-hop transfer first transfers the token from the origin spoke chain to the hub chain, where token balances are updated, and then triggers a second transfer to the final spoke chain.
```

Comment:
Nit: "origin spoke chain" might be confusing since the "origin" is the hub. It might be more clear to label the two spoke chains in the previous sentences as S_a and S_b, and then refer to them by their names.